apps/nextjs/src/app/dashboard/organizations/[org_slug]/create/_components/phases-info.tsx

```
@@ -0,0 +1,234 @@
+function BestOfSelect({ index, phase, setPhaseKeyValue }: PhaseFormProps) {
+  return (
+    <InputWrapper htmlFor={`phase-${index}-best-of`} label="Best of">
+      <Select
+        id={`phase-${index}-best-of`}
+        placeholder="Select Best Of..."
+        options={[
+          { value: "one", label: "1" },
+          { value: "three", label: "3" },
+        ]}
+        value={String(phase.bestOf)}
+        onValueChange={(value) => setPhaseKeyValue(index, "bestOf")(parseInt(value, 10))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)